MatrixKit/Categories/NSBundle+MatrixKit.m

```diff
@@ -15,16 +15,17 @@
  */
 
 #import "NSBundle+MatrixKit.h"
+#import "MXKViewController.h"
 
 @implementation NSBundle (MatrixKit)
 
 static NSString *customLocalizedStringTableName = nil;
 
 + (NSBundle*)mxk_assetsBundle
 {
-    NSString *bundleResourcePath = [NSBundle mainBundle].resourcePath;
-    NSString *assetPath = [bundleResourcePath stringByAppendingPathComponent:@"MatrixKitAssets.bundle"];
-    return [NSBundle bundleWithPath:assetPath];
+    NSBundle * bundle = [NSBundle bundleForClass:[MXKViewController class]];
```

Comment:
By the way, in the MatrixKitSample project, MatrixKit is not integrated as a Pod!
That's why the retrieved bundle is the main one.
To prevent the app from crash:
NSBundle * bundle = [NSBundle bundleForClass:[MXKViewController class]];
if (bundle == [NSBunde mainBundle]) {
  return bundle
}
NSURL * url = [bundle URLForResource:@"MatrixKit" withExtension:@"bundle"];
return [NSBundle bundleWithURL:url];